src/main/scala/io/confluent/examples/streams/algebird/CMSStoreChangeLogger.scala

```diff
@@ -33,7 +33,7 @@ class CMSStoreChangeLogger[K, V](val storeName: String,
                                  val partition: Int,
                                  val serialization: StateSerdes[K, V]) {
 
-  private val topic = ProcessorStateManager.storeChangelogTopic(context.applicationId, storeName, false)
+  private val topic = ProcessorStateManager.storeChangelogTopic(context.applicationId, storeName, null)
```

Comment:
any reason why we're passing `null` instead of `context.taskId().namedTopology()`?

Reply:
I don't think that this should be using the `namedTopology` stuff yet. We have not really finished that work 100% yet

Reply:
I am not familiar with the details of the upstream work. Did not even know that `context.taskId().namedTopology()` exists... Let me try this.